src/nss/nw_s1_aurafirec.nss

```diff
@@ -25,7 +25,7 @@ void main()
     //Declare major variables
     aoesDeclareMajorVariables();
     int nHD = GetHitDice(aoe.Creator);
-    int nDC = 10 + nHD/3;
+    int nDC = 7 + nHD/3;
```

Comment:
i think i originally changed this because anything <10 DC is a bit odd.

Any creature that used this probably will have at least 10 DC, and my change probably made it far harder to resist than it should've been.

Were I to re-visit this file I would probably add a check to make it min 10 DC rather than starting at 10 DC

Reply:
The ridiculous thing about this script is the CPP version seemingly didn't even use this variable for anything!

This was overwritten in the CPP update, I just copied the old version back in